### PR TITLE
find_minimum_prime

### PR DESCRIPTION
**Pull request by ahmedAdel202090**

### CODE REVIEW

**Comment by OmarHatem28** [comment]
1- your function should print the minimum prime number in the array, not the minimum number in the array
2- arr is defined globally so u can access it with out passing arguments to the function
3- add your function to case 18 // execute all
4- add this line after each printing statement u have, i.e in case 5 and in the function
System.out.println("\n=======================================");

**Comment by OmarHatem28** [comment]
revise the list i sent u in the first comment.

File: src/com/company/Main.java [hunk 1]
```diff
@@ -42,9 +41,12 @@ public static void main(String[] args) {
             case 17:
                 zeroIfNegative();
                 break;
+            case 5:
+            	find_smallest_prime();
+            	break;
             case 18: // Execute All
                 // Add Your Function Here
-                zeroIfNegative();
```
Comment (OmarHatem28) [inline src/com/company/Main.java]: this is an execute all function,
so literally it will execute all functions, why would u remove others function ?

Comment (OmarHatem28) [inline src/com/company/Main.java:47]: in the execute all case,
so literally it will execute all functions, why would u remove others function ?

Comment (ahmedAdel202090) [reply src/com/company/Main.java:47]: ok you can add it , i understand that this function like a template doesn't need in code

Comment (ahmedAdel202090) [reply src/com/company/Main.java:47]: i added it in last commit

**Review by OmarHatem28 — APPROVED** [review]

**Review by yosra99 — APPROVED** [review]

**Pull request closed without merging**